source/content/plugins-known-issues.md

```diff
@@ -745,26 +745,31 @@ ___
 
 ## [WP All Import / Export](http://www.wpallimport.com/)
 
-<ReviewDate date="2018-10-30" />
+<ReviewDate date="2020-06-15" />
 
 **Issue 1:** Large batch processes can fail if they take longer than the platform will allow. See [Timeouts on Pantheon](/timeouts) for more information.
 
 **Solution:** To avoid hitting a timeout, you can try:
+- Clean up temporary files
+- Lower the chunk size to less than 100
+- Cron Processing Time Limit should be set not more than 50 seconds to be safe with the 59 seconds platform PHP timeout.
+- Set the plugin to only process 1, this depends on how many post_meta's and custom functions are associated on every post imported, you can adjust it to a higher number that is optimal in your site if it will be a recurring import in order to speed it up but be reminded that it is a trial and error process which you need to experiment on works best from your end.
+- Instead of importing it as a one large file, it is best to set it up as reoccurring cron import as outlined [here]( https://www.wpallimport.com/documentation/recurring/cron/).
```

Comment:
Maybe rephrasing it from

`Instead of importing it as a one large file, it is best to set it up as reoccurring cron import as outlined [here]( https://www.wpallimport.com/documentation/recurring/cron/).`

to

`Instead of importing one large file, it is best to set it up as reoccurring cron import as outlined [here]( https://www.wpallimport.com/documentation/recurring/cron/).`

would be better